app/hooks/why_did_you_update.ts

```diff
@@ -29,8 +31,7 @@ export function useWhyDidYouUpdate(name: string, props: any) {
 
             // If changesObj not empty then output to console
             if (Object.keys(changesObj).length) {
-                // eslint-disable-next-line no-console
-                console.log('[why-did-you-update]', name, changesObj);
+                logInfo('[why-did-you-update]', name, changesObj);
```

Comment:
I don't see a reason to change this one.. thinking ahead about the framework I would not want nor expect this to log anything to the file system, so perhaps we can keep it as it was

Reply:
Or, what if we have an arg that will let us know that this log shouldn't go to the framework/sentry. If not, a separate logX function; e.g. logOnce ?

Reply:
I feel this is a really particular case. `useWhyDidYouUpdate` shouldn't even be in the codebase in general. I think it makes sense to make here the exception.